boards/shields/silabs_wifi_expansion_kit/silabs_wifi_expansion_kit_slexp8022a.overlay

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 grandcentrix GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/wifi/silabs_wf200.h>
+
+&raspberrypi_spi {
+    wfx200@0 {
+        status = "ok";
+        compatible = "silabs,wfx200";
+        reg = <0>;
+        label = "wifi0";
+        spi-max-frequency = <5200000>;
+
+        int-gpios = <&raspberrypi_header 20 GPIO_ACTIVE_HIGH>;
+        wake-gpios = <&raspberrypi_header 21 GPIO_ACTIVE_HIGH>;
+        reset-gpios = <&raspberrypi_header 22 GPIO_ACTIVE_LOW>;
+
+        pds = SILABS_WF200_PDS_WF200_EVAL;
```

Comment:
what's going on with this property?

Reply:
About that, i hoped documenting it in the device tree binding would be enough.
These are json-ish looking strings, which are sent during initialization to the WF200 chip. It configures quite a lot of properties, like which antenna port to use, how to configure its oscillator etc.. I really would like have these properties configurable in the device tree as individual properties, but its quite a bit of effort. I thought about generating these using macro magic from the devicetree or adding the PDS Template for the user to fill out and then generating the PDS with silabs pds_compress script. I don't like that property as its currently is, but i wanted to keep the PR small for now.